### DISCUSSION TITLE
Windows 10 compatibility issues

### DISCUSSION THREAD

**Discussion by smaranda123** [argoproj/argo-cd#6896] [Q&A] [answered]
Trying to install Argo CD CLI on Windows 10 64 bit and using the documentation provided, unfortunately ran across a few problems.

This is the short version of the steps that were taken:

1) Trying to install `argocd` using the documentation (https://argoproj.github.io/argo-cd/cli_installation/#windows)
`$url = "https://github.com/argoproj/argo-cd/releases/download/" + $version + "/argocd-windows-amd64.exe"
$output = "argocd.exe"
Invoke-WebRequest -Uri $url -OutFile $output`

2) Error encountered 
`This version of C:\ProgramData\chocolatey\bin\argo.exe is not compatible with the version of Windows you're running. Check your computer's system information and then contact the software publisher.`
   ->also see the png attached
![Capture](https://user-images.githubusercontent.com/88327319/127985472-581d3a6b-623c-4ce6-8145-3c3422b03ecf.PNG)

3) Tried with multiple versions, the result is the same.

4) Installed it using `choco` which grabs an old version, 1.6.1
   
   The only way and only that specific version that worked

5) Following up on the errors (step 2) led to this article 

https://docs.microsoft.com/en-us/windows/compatibility/ntvdm-and-16-bit-app-support

   And so, a dead end

**Comment by jopit** [comment]
The download command downloaded `argocd.exe`, but the error message is referring to `argo.exe`.  Are you actually running the executable that you dowloaded?

**Reply by smaranda123** [reply]
Yes, the `-OutFile` can be extracted as` argo.exe` and can be called with `argo` or can be extracted as  `argocd.exe` and can be called with `argocd`, the result is the same
   -> experienced these errors by running cmd

I have also tried using **Git Bash** and adding a path to the` .bashrc` file and the result is
`$ argocd`
`/c/Windows/system32/argocd: line 7: syntax error near unexpected token newline' `
`/c/Windows/system32/argocd: line 7: <!DOCTYPE html>' `

**Comment by smaranda123** [comment, marked as answer]
The problem was on my side, the download path that i was using was wrong, instead of `/download/` it was `/tags/`

This can be closed.
